frontend/src/tabs/zoom.pug

```diff
@@ -1,8 +1,14 @@
 #zoom
+  span.large-font Commits Panel
   .zoom__title
     .zoom_title--group.large-font
-      span(v-if="info.filterGroupSelection === 'groupByAuthors'") {{ info.user.displayName }} ({{ info.user.name }})
-      span(v-else) {{ info.user.repoName }}
+       span(v-if="info.filterGroupSelection === 'groupByAuthors'") {{ info.user.displayName }} ({{ info.user.name }})
+       a(
+       v-else,
```

Comment:
Meanwhile, it is always nice if you can reference how the indentation goes in other part of this file.
For example:
the line after `a(` should be extra indented with 2 space because it is the inside code of this `<a>`.